packages/react-core/src/components/Wizard/Wizard.tsx

```diff
@@ -85,6 +91,11 @@ export const Wizard = ({
     }
   }, [startIndex]);
 
+  const focusMainContentElement = () =>
+    setTimeout(() => {
+      wrapperRef?.current?.focus && wrapperRef.current.focus();
+    }, 0);
```

Comment:
Definitely curious about what scenario we could have that wouldn't include a main. If we went the `aria-live` route this would be necessary. However, maybe it's ok if we avoid using `aria-live` for now. I typically see live regions used for things like updates/status, alerts, logs, etc. Like [this article talks about](https://www.boia.org/blog/what-are-aria-live-regions), live regions typically aren't used for interactive content (and I imagine we'd expect wizard content to be interactive often). Based on our discussion, I agree that it probably makes sense to omit `aria-live` for now and rely on the announcement of the shifted focus.

Reply:
cc @mcoker

Reply:
> (Note that Core does not have a bunch of empty divs with display none for steps whose content is not currently rendered, so maybe the intent was that __main element should be static and __main-body dynamically renders step content? Curious as to whether we need these empty div's rendered in React at all.)

FWIW the wizard is built very similarly to a page, so `__main` was intended to mimic the page main container. In both cases, it's more of a static, structural element of the layout, so I would expect content within it (`__main-body` elements) to re-render, or possibly attributes and stuff on `__main` to change, but not the whole element to re-render... though I suppose that would be fine as long as it didn't cause layout shifts or anything?